packages/scoped-custom-element-registry/src/scoped-custom-element-registry.ts

```diff
@@ -467,7 +557,7 @@ if (!ShadowRoot.prototype.createElement) {
   ) {
     const internalsToHostMap = new WeakMap();
```

Comment:
Here and elsewhere, every `new Map`, `new WeakMap`, `new Set`, and `new WeakSet` call needs parameters or it's implicitly `any`

Reply:
Done